### PR TITLE
node name display in file viewer

### PR DESCRIPTION
**Pull request by manavortex**
Added more node names to 
- compiledData array (CSV): will use first array value
- animAnimSetEntry (.anim): will use nested animation name
- characterRecordId (.community): will use tweakDBEntry.toString()
- json: will use (in order) secondaryKey, femaleVariant, maleVariant

### CODE REVIEW

**Review by rfuzzo — DISMISSED** [review]
lgtm

File: WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs [hunk 1]
```diff
@@ -1816,8 +1816,26 @@ public void CalculateDescriptor()
 
         if (ResolvedData is IRedArray ary)
         {
+            if (PropertyName == "compiledData" && ary is [IRedArray { Count: 3 } childAry])
```
Comment (seberoth) [inline WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:1819]: What is this for?

Comment (manavortex) [reply WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:1819]: factory.csv entries (the comment probably got lost when I refactored)

File: WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs [hunk 2]
```diff
@@ -2322,7 +2346,8 @@ public void MoveChild(int index, ChunkViewModel item)
                 {
                     index--;
                 }
-                InsertChild(index, item.Data);
+
+                InsertChild(index, item?.Data);
```
[the remark this answers is not in the document]
Comment (manavortex) [reply WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:2350]: Rider disagreed hard, so I added the null-check

**Review by seberoth — APPROVED** [review]
lgtm

**Pull request merged**